### DISCUSSION TITLE
Example site config and title variable

### DISCUSSION THREAD

**Discussion by lszczygielek** [lxndrblz/anatole#246] [Q&A]
I would like to ask about the title variable in config.toml in exampleSite. Value is `title = "Website of Jane Doe"` and I made some tests and I can't trigger a scenario when this value is visible. Is it possible or maybe this variable should be removed from exampleSite?

**Comment by lxndrblz** [comment]
Hi @lszczygielek Thanks for bringing up this question.

I know, it is a bit confusing. In the past we had an issue where someone was asking the same question. I'll [link it](https://github.com/lxndrblz/anatole/issues/184#issuecomment-824343479) here for your convenience.

Basically, there Hugo uses a built-in parameter called title for generating the RSS feed and other default functionality. However, the theme uses a different title variable (the one under params) for controlling the output.

**Reply by lszczygielek** [reply]
Thank you for your quick answer.
